internal_ws/ykcompile/src/lib.rs

```diff
@@ -288,7 +290,7 @@ pub extern "sysv64" fn bh_new_vec() -> *mut Vec<FrameInfo> {
 
 /// Pushes a new `FrameInfo` instance onto the vector behind the pointer `vptr`. The `FrameInfo`
```

Comment:
Since we know that the pointer is really a `Box` -- and we rely on that by calling `Box::from_raw`! -- we should document that this pointer *must* have been created with `Box::new`. Or, alternatively, we could stop calling `Box::from_raw`, use @bjorn3's `&mut *` suggestion, and then we don't need to specify that the vector must have been created with `Box`. I tend to prefer the latter approach as it's one thing fewer for our future selves to get wrong.

Reply:
Fixed in 2c47787.